packages/app/src/app/pages/Dashboard/Content/routes/Recent/index.tsx

```diff
@@ -65,29 +68,37 @@ export const Recent = () => {
         .slice(0, 12);
 
   const pageType: PageTypes = 'recent';
+  const showUpgradeBanner = isFree && isTeamSpace;
   const isEmpty = !dataIsLoading && items.length === 0;
 
+  if (isEmpty) {
+    return <EmptyRecent showUpgradeBanner={showUpgradeBanner} />;
+  }
+
   return (
-    <SelectionProvider
-      activeTeamId={activeTeamId}
-      page={pageType}
-      items={items}
-    >
+    <SelectionProvider activeTeamId={activeTeam} page={pageType} items={items}>
       <Helmet>
         <title>Recent - CodeSandbox</title>
       </Helmet>
-      {isFree && isTeamSpace && <UpgradeBanner teamId={activeTeamId} />}
+      {showUpgradeBanner && (
+        <Element
+          css={{
+            width: `calc(100% - ${2 * GUTTER}px)`,
+            maxWidth: `calc(${GRID_MAX_WIDTH}px - 2 * ${GUTTER}px)`,
+            margin: '0 auto',
+            paddingBottom: '48px', // Because the stack gap is 0, it overrides the margin bottom so we need to use padding.
+          }}
+        >
+          <UpgradeBanner teamId={activeTeam} />
+        </Element>
+      )}
       <Header
-        title={isEmpty ? "Let's start building" : 'Recent'}
-        activeTeam={activeTeamId}
+        title="Recent"
+        activeTeam={activeTeam}
         loading={dataIsLoading}
-        showViewOptions={!isEmpty}
+        showViewOptions
```

Comment:
Header still needs some padding bottom to avoid content scrolling immediately under the text / view toggle button

<img width="1525" alt="image" src="https://user-images.githubusercontent.com/9945366/209091349-55dafd97-541a-44d1-aa52-929196b351a0.png">

Reply:
Fixed in: [fa55774](https://github.com/codesandbox/codesandbox-client/pull/7264/commits/fa557744b819cafb4047b063b06f41d414fe76dc)

![57t7wm-3000 preview csb app_dashboard_repositories_workspace=b5123bcc-da66-446f-88e2-5b89c9c37d27](https://user-images.githubusercontent.com/24959348/209163539-9ae1cb8a-4353-4d3e-ae28-347d3e1e6f98.png)